driver/src/contracts/stablex_contract.rs

```diff
@@ -8,23 +8,26 @@ use web3::types::{H160, U128, U256};
 
 use dfusion_core::models::{AccountState, Order, Solution};
 
-use crate::error::DriverError;
-
 use super::base_contract::BaseContract;
 use super::stablex_auction_element::StableXAuctionElement;
+use crate::error::DriverError;
 
 type Result<T> = std::result::Result<T, DriverError>;
 
 pub const AUCTION_ELEMENT_WIDTH: usize = 112;
+lazy_static! {
+    // In the BatchExchange smart contract, the objective value will be multiplied by
+    // 1 + IMPROVEMENT_DENOMINATOR = 101. Hence, the maximal possible objective value is:
+    static ref MAX_OBJECTIVE_VALUE: U256 = U256::max_value() / (U256::from_dec_str("101").unwrap());
```

Comment:
I would use U256::from(101) here, so there is no need to parse a string or unwrap.